src/dialogs/permissions.tsx

```diff
@@ -258,9 +275,9 @@ const EditPermissionsModal = ({ dialogResult, selected, path }) => {
           variant={ModalVariant.small}
           /* Translators: $0 represents a filename */
           title={fmt_to_fragments(_("$0 permissions"), <b>{selected.name}</b>)}
-          description={inode_types[selected.type] || "Unknown type"}
+          description={(selected.type) ? inode_types[selected.type] : "Unknown type"}
           isOpen
-          onClose={dialogResult.resolve}
+          onClose={() => dialogResult.resolve()}
```

Comment:
Hmm why is this not happy for typing?

Reply:
`Type '(value: void) => void' is not assignable to type '(event: KeyboardEvent | MouseEvent<Element, MouseEvent>) => void'.